library/src/webapp/js/portal/portal.all.sites.js

```diff
@@ -59,7 +42,7 @@ portal.allsites.setAllOrNoneStarStates = function () {
 
   sidebar.querySelectorAll(".favorites-select-all-none").forEach(function (selectAllNone) {
 
-    const termContainer = selectAllNone.closest(".fav-sites-term");
+    const termContainer = selectAllNone.closest(".card-body");
```

Comment:
`card-body` is a bootstrap-5 class. Are you sure this won't interfere with other cards?

Reply:
I'm sure. There is a card-body right above it. Do you think "fav-sites-header" would be better? It might be, perhaps.

Reply:
Yes, anything but b-5 classes

Reply:
Done